pkg/plugin/plugin.go

```diff
@@ -74,6 +76,12 @@ func (p *plugin) Validate(ctx context.Context, new, old *api.OpenShiftManagedClu
 		}
 	}
 
+	if old != nil && !new.Properties.RefreshCluster {
```

Comment:
above this line, suggest returning early if `len(errs) > 0`

Reply:
just tried, makes the unit tests horrid as the configs have to be perfect to catch the error i want to catch.